src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/integ_test/CpuMetricsIT.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright <2020> Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.integ_test;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.integ_test.json.JsonResponseData;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.integ_test.json.JsonResponseField.Type.Constants;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.integ_test.json.JsonResponseNode;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.metrics.AllMetrics.OSMetrics;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CpuMetricsIT extends MetricCollectorIntegTestBase {
+  private List<String> nodeIDs;
+
+  @Before
+  public void init() throws Exception {
+    nodeIDs = getNodeID();
+  }
+
+  @Test
+  public void checkCPUUtilization() throws Exception {
+    //read metric from local node
+    List<JsonResponseNode> responseNodeList =
+        readMetric(PERFORMANCE_ANALYZER_BASE_ENDPOINT + "/metrics/?metrics=CPU_Utilization&agg=sum");
```

Comment:
nit: can we create path as constant variable?

Reply:
this endpoint is only referenced once here. and I have defined a const var for the common path that can be used by other endpoint. Do we still need to define a seperate static var here ?